src/ontology/uberon-edit.obo

```diff
@@ -224756,6 +224756,14 @@ intersection_of: UBERON:0001137 ! dorsum
 intersection_of: in_right_side_of UBERON:0000468 ! multicellular organism
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: UBERON:8000008
+name: proximal musculature of limb
+def: "Musculature of the limb that is on the proximal side of the limb musculature." [http://orcid.org/0000-0001-5208-3432]
+intersection_of: UBERON:0004480 ! musculature of limb
+intersection_of: in_proximal_side_of UBERON:0007271 ! appendage musculature
```

Comment:
Two things to always mentally check before making an equivalence axiom

 1. does this correspond to a DP. If not, proceed carefully
 2. does the text definition match the logical definition? And does the logical definition match the name? See https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/ -- if not, do not make a logical definition

Reply:
axiomatization using the musculature terms is actually really hard. See #2021 

if musculature of limb is the entire musculature of limb then this logical definition then this class is unsatisfiable since you can't make a part from a whole

if instead musculature of X means some portion of X then the class doesn't do what you want it to since this could refer to e.g. the proximal side of the toe muscles.

I think this is a very low priority term, see my comments on the ticket. not every hpo class needs a logical def. They need text definitions more! I suggest closing this

Reply:
sounds good, I'll close this and the similar ticket. Thanks for your feedback, Chris!